### PR TITLE
O in visual is supported in panels with the "i" key ##panels

### PR DESCRIPTION
**Pull request by Vane11ope**

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -808,16 +808,47 @@ bool __check_panel_type(RPanel *panel, const char *type) {
...
 				strcmp (panel->model->cmd, PANEL_CMD_DECOMPILER_O) &&
 				strcmp (panel->model->cmd, PANEL_CMD_DISASMSUMMARY)) {
+			free (tmp);
+			return true;
+		}
+		free (tmp);
+		return false;
+	}
+	if (!strcmp (type, PANEL_CMD_STACK)) {
+		if (!strcmp (tmp, PANEL_CMD_STACK)) {
```
Comment (ret2libc) [inline libr/core/panels.c]: what about removing the `if` and just doing 
```C
free (tmp);
return !strcmp (tmp, PANEL_CMD_STACK);
```

File: libr/core/panels.c [hunk 2]
```diff
@@ -808,16 +808,47 @@ bool __check_panel_type(RPanel *panel, const char *type) {
...
+		return false;
+	}
 	int len = strlen (type);
 	if (!strcmp (type, PANEL_CMD_DISASSEMBLY)) {
-		if (!strncmp (panel->model->cmd, type, len) &&
+		if (!strncmp (tmp, type, len) &&
 				strcmp (panel->model->cmd, PANEL_CMD_DECOMPILER) &&
 				strcmp (panel->model->cmd, PANEL_CMD_DECOMPILER_O) &&
 				strcmp (panel->model->cmd, PANEL_CMD_DISASMSUMMARY)) {
+			free (tmp);
```
Comment (radare) [inline libr/core/panels.c]: i would rather use a bool variable to specify the return and just have the free+return once

File: libr/core/panels.c [hunk 3]
```diff
@@ -6131,7 +6175,15 @@ void __rotate_entropy_h_cb(void *user, bool rev) {
 
 void __rotate_hexdump_cb (void *user, bool rev) {
 	RCore *core = (RCore *)user;
-	__rotate_panel_cmds (core, hexdump_rotate, COUNT (hexdump_rotate), "px", rev);
+	RPanel *p = __get_cur_panel (core->panels);
+
+	if (rev) {
+		p->model->rotate--;
```
Comment (radare) [inline libr/core/panels.c]: p->model->rotate += (ret?-1:1);

**Comment by radare** [comment]
I'm merging without doing those cleanups, changes look good to me, and we can always improve the LOCs and code readability. thanks!

**Pull request merged**